cmd/copy.go

```diff
@@ -180,6 +180,9 @@ type rawCopyCmdArgs struct {
 	rehydratePriority string
 	// The priority setting can be changed from Standard to High by calling Set Blob Tier with this header set to High and setting x-ms-access-tier to the same value as previously set. The priority setting cannot be lowered from High to Standard.
 	trailingDot string
+
+	// when specified, the dst blob with uncommitted blocks is deleted
+	deleteUncommittedBlocks bool
```

Comment:
This is a misnomer - we'll have to make it clear that we're deleting the blob itself and not just un-committted blocks

Reply:
Updated!